### ISSUE TITLE
Make a release

### ISSUE THREAD

**Issue by KeepBotting** [sm00th/bitlbee-discord#184]
Hi, due to policy and how the build system works, packages for Void Linux are generally only accepted if they correspond to a tagged release.

Thus, the Void [template](https://github.com/void-linux/void-packages/blob/4a00d41d3d4757f443b56899010f420f718a4672/srcpkgs/bitlbee-discord/template) for `bitlbee-discord` is somewhat out of date, and the package built will be broken as of a few days ago due to Discord's change of how nonces are read from the client.

Please tag a release that includes this bugfix so the package may be updated in the distribution, instead of a broken version being held.

Thank you for maintaining this software, I use it on a constant basis every day.

**Comment by sm00th** [comment]
Right, the nonce issue is, indeed, big enough to trigger a release. I've tagged 0.4.2, guess this should be enough.

**Comment by anarcat** [comment]
i hate to bring up an old issue like this, but it seems there's a few critical fixes pending in the backlog here, maybe it's time to make a release again?
